docs/api-reference.md

```
@@ -423,28 +423,15 @@ Clear all event log entries.
-Basic health check endpoint.
-
-**Response:**
-```json
-{
-  "status": "healthy",
-  "timestamp": "2025-09-04T12:00:00Z"
-}
-```
+## Application Information
 
 ### GET `/api/version`
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)